### ISSUE TITLE
C# compiler is out of date

### ISSUE THREAD

**Issue by robsimmons** [frankpfenning/C0#45]
Low priority, but the cc0-to-C# should eventually be looked at, and the regression (make checkcsharp) should be brought back up-to-date.

**Comment by frankpfenning** [comment]
Yes, this is low priority.  The reason we built this is to see if we can tie in tools
for static verification, test case generation, etc. that have been built for C#.
If someone takes up this line of research again (at the moment there are no
plans), then whoever it is can bring C# up to date.  The language spec keeps
a history log, so language changes are documented.

We should mark this in the README, and also mark the analysis features
as "unreleased".  Right now, students can see "static analysis options" which
should be hidden, or at least clearly flagged as research in progress.

**Comment by robsimmons** [comment]
I just commented out --static-check and --verif-check in r250... this means that nullness checking is also out? What's the call there for s12?

**Comment by frankpfenning** [comment]
I'll take a look at null checking today.

The lack of realistic tests was one of the issues, but that is
resolved now.
